server/plugin/command.go

```diff
@@ -85,6 +85,46 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) muteGithubUser(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	mutedUsernameBytes, _ := p.API.KVGet(userInfo.UserID + "-muted-users")
+	mutedUsernames := string(mutedUsernameBytes)
+	if len(mutedUsernames) > 0 {
+		// , is a character not allowed in github usernames so we can split on them
+		mutedUsernames = mutedUsernames + "," + strings.Join(parameters, ",")
+	} else {
+		mutedUsernames = strings.Join(parameters, ",")
+	}
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(mutedUsernames)); err != nil {
+		return "Error occurred saving list of muted users"
+	}
+	return mutedUsernames
+}
+
+// Returns the elements in a, that are not in b
+func arrayDifference(a, b []string) []string {
+	mb := make(map[string]struct{}, len(b))
+	for _, x := range b {
+		mb[x] = struct{}{}
+	}
+	var diff []string
+	for _, x := range a {
+		if _, found := mb[x]; !found {
+			diff = append(diff, x)
+		}
+	}
+	return diff
+}
```

Comment:
While the code looks fine, I'm wondering if we should simplify the design and only allow passing one username via `mute-users` similar how a user can only subscribe to one repo at a time.

Reply:
I personally think that the ability to mute and unmute multiple at once is nice, but I don't mind (and actually thought about) making it singular. That being said, the logic of this function wouldn't change a whole lot, because we'd still be parsing the KV stored values as a CSV array. Let me know if you'd like me to change it!